jobboard/tests/view_tests.py

```diff
@@ -0,0 +1,76 @@
+import pytest
+from pytest_django.asserts import assertTemplateUsed
+from jobboard.models import Job
+from student.models import Major
+from datetime import date, timedelta
+from django.contrib.auth.models import User
+from student.models import Student, EducationalInstitution
+from recruiter.models import Recruiter
+
+# The tests use the data that is already in the db from the migration files
+
+
+@pytest.fixture
+def example_institution_a():
```

Comment:
Both files use the exact same fixtures, maybe you can make one file import the fixtures from the other to avoid code duplication?

Reply:
Done